functions/venue.js

```diff
@@ -193,7 +193,8 @@ const createVenueData = (data, context) => {
     owners,
     profile_questions: data.profile_questions,
     placement: { ...data.placement, state: PlacementState.SelfPlaced },
-    showLiveSchedule: data.showLiveSchedule ? data.showLiveSchedule : false,
+    // @debt find a way to share src/settings with backend functions, then use DEFAULT_SHOW_SCHEDULE here
+    showSchedule: data.showSchedule ? data.showSchedule : true,
```

Comment:
This can be simplified to:
```suggestion
    showSchedule: data.showSchedule || true,
```

Reply:
Something doesn't feel right here.

What happens if `data.showSchedule` is explicitly set to `false`?
Will it continue being such after `false || true`?

I think if `true` is supposed to be a default in case the value is missing 
i.e. is `null`/`undefined`, 
the operator should be `??` or the equivalent `?:` check for older JS

Reply:
NVM, found the resolution to it https://github.com/sparkletown/sparkle/pull/1657#discussion_r659145693 although an explicit test for boolean... `typeof new Boolean()` all's well I guess, no1 should be making these wrapper objects anyway